### PR TITLE
fix(env): import env variables from one file

### PR DESCRIPTION
**Pull request by karansapolia**
## PR Checklist
Please check that your PR fulfills the following requirements:

- [x] The commit message follows [our guidelines](https://github.com/pesto-students/batch-9-Lumen/wiki/Commit-message-convention)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] GIFs/Videos/Images added (only for frontend changes)

## PR Type
What kind of change does this PR introduce? 

- [x] Bugfix
- [x] Feature
- [x] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Infrastructure changes
- [ ] Other... Please describe:

##Issue Number: 
#73 

## Does this PR introduce a breaking change?
- [ ] Yes
- [x] No

## Other information
Blocks #72

### CODE REVIEW

**Review by nimish-gupta — APPROVED** [review]

**Review by piyushagwl007 — APPROVED** [review]

File: server/src/configs/env.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const envConfig = {
+  port: process.env.PORT || 3000,
+};
+
+export default Object.freeze(envConfig);
```
Comment (nihalanisumit) [inline server/src/configs/env.js]: jhjhv

File: server/src/configs/env.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const envConfig = {
```
Comment (nihalanisumit) [inline server/src/configs/env.js]: why we are not using .env file? Also please add this file to .gitignore

**Review by nihalanisumit — COMMENTED** [review]
use .env

**Review by nihalanisumit — COMMENTED** [review]
LGTM

**Comment by karansapolia** [comment]
> LGTM

Request you to mark it as approved :smile:

**Comment by karansapolia** [comment]
> use .env

According to our discussions about .env and sharing of variable names across the team, If you could suggest a way of working on this, that would be very helpful :smile: We could edit these files later in a 'feat' or 'fix' PR and collate all .env variables to one file

**Review by nihalanisumit — APPROVED** [review]
LGTM!

**Pull request merged**